at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -110,9 +104,9 @@ class ScanVerbHandler extends AbstractVerbHandler {
   /// @param - atConnectionMetadata: Metadata of the inbound connection
   /// @param - List<String>: List of keys from the secondary persistent store
   /// @return - String: Returns the keys of current atsign
-  String _getLocalKeys(
+  List<String?> _getLocalKeys(
```

Comment:
Can you improve the documentation of this method please? Its purpose is not clear to me. "Returns the current atsign keys" doesn't tell me intent, and the code is not too obvious either. Also the parameter descriptions don't tell me anything useful

Reply:
So sorry about that @gkc, I didn't notice that. I will improve it.

Reply:
Thanks Minnu. (PS No need for apologies!)